packages/connect-explorer/README.md

```diff
@@ -9,8 +9,12 @@ To run the explorer locally:
 
 `yarn workspace @trezor/connect-explorer dev` - runs dev server on port 8088
 
-When `connect-exporer` is run locally, WebUSB is disabled because [@trezor/connect-popup](../connect-popup) runs on a different domain and Chrome does not allow `navigator.usb.requestDevice` calls between cross-site elements since v72. Further investigation needed to see if we can fix that. Until then, run [Bridge](https://suite.trezor.io/web/bridge/) to connect to Trezor.
+### Examples
```

Comment:
The semantics here do not seam correct to me, since the first element title `# @trezor/connect-explorer` contains only one `#`.
Therefore element bellow should contain only 2 `#` like:

`## Examples` and `## Webusb`.